src/main/java/pl/isa/biblioteka/booksClass/BorrowBooks.java

```diff
@@ -10,70 +10,44 @@ public class BorrowBooks {
     public static Scanner scanner = new Scanner(System.in);
     public static BooksEdit booksEdit = new BooksEdit();
 
-    public static void addBookToPerson() {
+    public static void addBookToPerson(List<Book> personBooks) {
         List<Book> booksList = BooksEdit.booksList;
-        List<Person> users = Users.getUsers();
-        String firstName = getFirstName();
-        String lastName = getLastName();
-        boolean findUser = false;
+        System.out.println("Podaj tytuł książki do wypożyczenia: ");
+        String bookTitle = scanner.nextLine();
         boolean findBook = false;
-        for (Person user : users) {
-            if (firstName.equalsIgnoreCase(user.getFirstName()) && lastName.equalsIgnoreCase(user.getSecondName())) {
-                System.out.println("Mamy Cię w naszej bazie ;)");
-                System.out.println("Podaj tytuł książki do wypożyczenia: ");
-                String bookTitle = scanner.nextLine();
-                for (Book book : booksList) {
-                    if (book.getTitle().equalsIgnoreCase(bookTitle) && book.isState()) {
-                        book.setState(false);
-                        user.personBooks.add(book);
-                        System.out.println("Książka została wypożyczona");
-                        findBook = true;
-                        break;
-                    }
-                }
-                if (!findBook) {
-                    System.out.println("Nie posiadamy książki o podanym tytule");
-                }
-                findUser = true;
+        for (Book book : booksList) {
+            if (book.getTitle().equalsIgnoreCase(bookTitle) && book.isState()) {
+                book.setState(false);
+                personBooks.add(book);
```

Comment:
To teraz trochę skomplikuje, bo zamiast przekazywać listę książek, przekazuj nazwę, jaką wprowadził użytkownik (patrz linia ze Scannerem). Natomiast jeśli mamy zalogowanego użytkownika, to mamy też zapewne metodę, która potrafi nam go zwrócić, a wtedy już prosta droga do tego, żeby przypisać mu książki